graphics/dcmtk/Portfile

```diff
@@ -38,12 +38,12 @@ checksums \
 compiler.cxx_standard   2011
 
 patchfiles              install_path_fix.patch \
-                        install_path_fix_part2.patch
+                        install_path_fix_part2.patch \
+                        patch-fix-powerpc.diff
```

Comment:
it this fixes stuff for PPC please apply the patch only there.

Reply:
@reneeotten It fixes PowerPC, but it is just the correct code and does not affect other archs at all. It is also an absolutely minimal patch. It should be used unconditionally.

Reply:
It's less difficult for us to review conditional patches. If it's conditional, then we don't have to actually carefully understand what it does on a modern platform.

Reply:
We are talking about a patch which changes a _single line_ in 3 instances.
```
-#if defined(__APPLE__) && !defined(__aarch64__)
+#if defined(__APPLE__) && !defined(__aarch64__) && !defined(__POWERPC__)
```
Literally. I do not know what can be more transparent :)

It is preferable to have unproblematic patches unconditional, because then less breakages happen. We had many cases when patches are not rebased and then fail to apply.

Reply:
@pmetzger I mean, I will do it if you really insist, but maybe take a look at the patch first?
We talk about this one: https://github.com/macports/macports-ports/blob/cc21ec4f28d993feee5f97d46386dbb5a7164492/graphics/dcmtk/files/patch-fix-powerpc.diff